src/openforms/registrations/contrib/email/tests/test_backend.py

```diff
@@ -451,7 +475,33 @@ def test_submission_with_email_backend_export_csv_xlsx(self):
         self.assertEqual(
             xlsx_export[0], f"{submission.form.admin_name} - submission.xlsx"
         )
-        self.assertEqual(xlsx_export[1], create_submission_export(qs).export("xlsx"))
+
+        data = tablib.Dataset()
+        binary_excel = xlsx_export[1]
+        data.load(binary_excel)
+
+        self.assertEqual(
+            data.headers,
+            [
+                "Formuliernaam",
+                "Inzendingdatum",
+                "someField",
+                "someList",
+                "attachment1",
+                "attachment2",
+            ],
+        )
+        self.assertEqual(
+            list(data._data[0]),
+            [
+                "MyInternalName",
+                datetime(2021, 1, 1, 12, 0),
+                "value0",
+                "['value1', 'value2']",
+                None,
+                None,
+            ],
+        )
```

Comment:
I would have been happy with asserting that whatever `generate_minimal_setup` came up with, was in the loaded data. Maybe even happier; as it focusses on the round-trip aspect of import after export.

This fixes the test too, so it suffices.